test/recipes/80-test_pkcs12.t

```diff
@@ -233,4 +233,7 @@ ok(run(test(["pkcs12_api_test",
              "-has-cert", 1,
              ])), "Test pkcs12_parse()");
 
+ok(run(test(["pkcs12_api_test",
+            ])), "Test pkcs12_create_ex2_test()");
```

Comment:
This doesn't seem necessary since it looks to me like the new tests will already be invoked by the existing runs of pkcs12_api_test